CHANGELOG.md

```diff
@@ -1,27 +1,34 @@
-﻿# Change Log
-
-<!--## Unreleased
-- Support GetCurrentWindowHandle and GetWindowHandles
-- Add driver option --silent to disable logging
--->
-
-
## v1.5.0
-
+﻿# Change Log
+
+<!--## Unreleased
+- Support GetCurrentWindowHandle and GetWindowHandles
+- Add driver option --silent to disable logging
+-->
+
+
+## v1.5.1
```

Comment:
Consider merging 1.5.0 and 1.5.1, we never release 1.5.0 so no need to have 2 versions